test/party/get-parties-by-search-text.api-test.ts

```diff
@@ -141,10 +152,10 @@ describe('GET /parties?searchText={searchText}', () => {
     expect(body).toStrictEqual({
       statusCode: 400,
       error: 'Bad Request',
-      message: ['searchText must be longer than or equal to 3 characters', 'searchText must match /^\\s*\\S+\\s*$/ regular expression'],
+      message: ['searchText must be longer than or equal to 3 characters', 'searchText must match /.*\\S.*/ regular expression'],
```

Comment:
As discussed on call, we may be able to remove the regex altogether if the all whitespaces case is caught by the length validation anyway.

Reply:
Done 👍